sydent/http/servlets/store_invite_servlet.py

```diff
@@ -133,6 +133,9 @@ def render_POST(self, request: Request) -> JsonDict:
 
         substitutions["ephemeral_private_key"] = ephemeralPrivateKeyBase64
         if substitutions["room_name"] != "":
+            if len(substitutions["room_name"]) > 30:
+                substitutions["room_name"] = substitutions["room_name"][:25] + "…"
```

Comment:
Unicode is terrible and we should note that we can may truncate in the middle of a grapheme here, eg. when there are multi-code point emoji or letters with combining diacritics.

Reply:
Good point! I don't really care about that right now though, but we should clean it up at some point